src/d3a/models/strategy/load_hours.py

```diff
@@ -112,6 +113,8 @@ def event_activate(self):
                             for day in range(self.area.config.sim_duration.days + 1)}
         self._simulation_start_timestamp = self.area.now
         self.assign_energy_requirement(self.avg_power_W)
+        self.energy_requirement_history_kWh = dict((k, v / 1000.)
```

Comment:
Is this different from self.state.desired_energy_Wh (member of LoadState class) ? If not, then it is better to reuse the member from the state class, in order to not add another class member.

Reply:
Yes, you are right, can not remember the reason for adding this here. Will try to remove it.